### PR TITLE
Fix test: test_region_heartbeat_report_approximate_size

### PR DESCRIPTION
**Pull request by Little-Wallace**
### What problem does this PR solve?
Fix unstable test test_region_heartbeat_report_approximate_size
Issue Number: close #xxx 

Problem Summary:

### What is changed and how it works?
I crease the data size of written keys and batch them into one raft-entry.

What's Changed:

### Related changes

- PR to update `pingcap/docs`/`pingcap/docs-cn`:
- PR to update `pingcap/tidb-ansible`:
- Need to cherry-pick to the release branch

### Check List 

Tests 

- Unit test
- Integration test
- Manual test (add detailed scripts or steps below)
- No code

Side effects

- Performance regression
    - Consumes more CPU
    - Consumes more MEM
- Breaking backward compatibility

### Release note 
```
None
```

### CODE REVIEW

File: tests/failpoints/cases/test_split_region.rs [hunk 1]
```diff
@@ -659,3 +660,69 @@ fn test_split_duplicated_batch() {
     rx2.recv_timeout(Duration::from_secs(3)).unwrap();
     must_get_equal(&cluster.get_engine(3), b"k11", b"v11");
 }
+
+#[test]
+fn test_report_approximate_size_after_split_check() {
```
Comment (BusyJay) [inline tests/failpoints/cases/test_split_region.rs:665]: Please add some comment describing the purpose of the case.

**Review by BusyJay — APPROVED** [review]
Rest LGTM

**Review by Connor1996 — COMMENTED** [review]
Can you describe why the test is flaky in the PR.

**Review by Connor1996 — APPROVED** [review]
LGTM

**Review by zhouqiang-cl — APPROVED** [review]
LGTM

**Pull request merged**